test/extended/builds/subscription_content.go

```diff
@@ -91,3 +94,22 @@ var _ = g.Describe("[sig-builds][Feature:Builds][subscription-content] builds in
 	})
 
 })
+
+// Read and Convert the BuildConfig to use the architecture of the cluster
+func ReadBuildConfig(path string, oc *exutil.CLI) string {
+	data, _ := ioutil.ReadFile(path)
+	arch := ClusterArchitecture(oc)
+	return strings.Replace(string(data[:]), "$arch", arch, -1)
+}
+
+// Get the cluster architecture
+func ClusterArchitecture(oc *exutil.CLI) (arch string) {
+	output, err := oc.WithoutNamespace().AsAdmin().Run("get").Args("nodes", "-o=jsonpath={.items[*].metadata.labels.kubernetes\\.io/arch}").Output()
```

Comment:
Kubernetes uses `GOARCH` values to populate this [label](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-arch). I don't think this aligns with RHEL arch names.

I believe we want the value from the `arch` command here.